apps/backend/src/modules/users/user.repository.ts

```diff
@@ -101,8 +101,8 @@ class UserRepository implements Repository {
 					createdAt: user.createdAt,
 					email: user.email,
 					hasAnsweredOnboardingQuestions:
-						user.onboardingAnswers.length > ZERO_INDEX,
-					hasAnsweredQuizQuestions: user.quizAnswers.length > ZERO_INDEX,
+						user.onboardingAnswers.length > FIRST_ITEM_INDEX,
+					hasAnsweredQuizQuestions: user.quizAnswers.length > FIRST_ITEM_INDEX,
```

Comment:
Again, the name has to make sense. You are now comparing length and index, it doesn't make sense